crosslink/README.md

```diff
@@ -0,0 +1,128 @@
+# Crosslink
+
+Crosslink is a tool to assist in managing go repositories that contain multiple
+intra-reposistory go.mod files. Crosslink automatically scans and inserts
+replace statements for direct and transitive intra-repository dependencies.
+Crosslink also contains functionality to remove any extra replace statements
+that are no longer required within reason (see below).
+
+## Rules
+
+Crosslink makes certain assumptions about your repository. Due to these
+assumptions the tool maintains contstraints to avoid any undesirable changes.
+
+1. By default all crosslink actions are non destructive unless specific command
+    flags are provided.
+2. Crosslink will only work with modules that fall under the root module
+    namespace.
+   - For example `example.com/crosslink/foo` falls under the
+    `example.com/crosslink` namespace but `example.com/bar` does not.
+   - Root module namespace is defined either by the module that exists in the
+    provided `--root` flag directory or the `go.mod` file located at highest
+    level of the repository.
+3. Crosslink does not maintain or include version numbers in replace
+   statements. Replace statements are always inserted or overwritten with no
+    version numbers.
+4. Crosslink allows users to `exclude` modules. Exclude means that crosslink
+   will not perform any replace or pruning operations where the
+   `old path == exclude path`. Operations will still be performed inside
+   modules where `module path == exclude path`.
+
+`Note:` Crosslink was developed for use in the OpenTelemetry organization.
+See [opentelemetry-go](https://github.com/open-telemetry/opentelemetry-go)
+and
+[opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib)
+for working examples. If you exerperience a use case that crosslink fails
+too handle properly please open an issue (or even a PR!) highlighting
+the discrepancy.
+
+## Usage
+
+### Latest Release
+
+To utilize the latest release of crosslink clone and build the repository and
+use the commands listed below.
```

Comment:
Hmm, I can't figure out how to suggest a code block, but can we recommend installing with `go`?
```shell
go install go.opentelemetry.io/build-tools/crosslink
```

Reply:
Will add